modules.js

```diff
@@ -6,183 +6,173 @@ const { Client } = require("@notionhq/client");
 
 require("dotenv").config();
 
-if (!(process.env.NOTION_API_KEY && process.env.DATABASE_ID)) {
-    throw new Error("Please fill in your API key and database ID in repl.it");
-}
```

Comment:
I moved this to the main function when starting the app. It seemed like an app critical piece that fit better over there.